include/mgp.hpp

```diff
@@ -645,6 +645,8 @@ class Node {
   Relationships OutRelationships() const;
   /// @brief Adds a label to the node.
   void AddLabel(const std::string_view label);
+  /// @brief Removes a label from the node.
```

Comment:
Can you just add spacing here between functions. 
You can check this Ivan's PR: https://github.com/memgraph/memgraph/pull/1103/files

Reply:
Format also rest of the Relationships class so we are consistent